org.wxhexeditor.wxHexEditor.json

```diff
@@ -0,0 +1,74 @@
+{
+    "app-id": "org.wxhexeditor.wxHexEditor",
+    "runtime": "org.gnome.Platform",
```

Comment:
Did you try `org.freedesktop.Platform//18.08`? Just curious if this actually needs anything from GNOME.